angular-standalone/official/blank/src/app/home/home.page.ts

```diff
@@ -6,7 +7,7 @@ import { IonicModule } from '@ionic/angular';
   templateUrl: 'home.page.html',
   styleUrls: ['home.page.scss'],
   standalone: true,
-  imports: [IonicModule],
+  imports: [CommonModule, IonicModule],
```

Comment:
Is there an issue you are experiencing without the `CommonModule`? The `HomePage` template isn't using declarations from `CommonModule`, so it shouldn't be necessary for the starter example.

Reply:
No issue...it's just that given this is intended to be starter project, the missing `CommonModule` could trip up novice users that aren't aware that things like `ngIf` and `ngFor` aren't available without it. It may also be confusing that it is part of some of the starter templates but not this one. This PR is intended to help avoid potential confusion/frustration only.

Reply:
I think the idea is that we shouldn’t be including things not needed to get the page to render. If a user needs `ngIf`, they should have to import it. Vs including CommonModule which could impact bundle size